src/Database/index.js

```diff
@@ -61,14 +63,27 @@ export default class Database {
     })
     await this.adapter.batch(operations)
 
+    const sortedOperations = []
```

Comment:
Not sure how using groupBy from rambdax would help. I would need to put the collection-name as the group-name but then lost the reference to collection. I could not find a way to improve this.